.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/libsdl-org/SDL
 [submodule "build/submodules/Vulkan-Headers"]
 	path = build/submodules/Vulkan-Headers
-	url = https://github.com/KhronosGroup/Vulkan-Headers
+	url = https://github.com/KhronosGroup/Vulkan-Headers
```

Comment:
What has changed here?

Reply:
probably line endings idk, innocent in any case

Reply:
oh i know now, fixed

```suggestion
	url = https://github.com/KhronosGroup/Vulkan-Headers

```